src/templates/Navigator/MainNavigatorComponent/Readme.md

```diff
@@ -0,0 +1,5 @@
+MainNavigatorComponent Example:
+
+    <MainNavigatorComponent>
```

Comment:
Because we don't want to use a general name that might conflict with another component.